runtime/drivers/duckdb/connectors.go

```diff
@@ -68,7 +68,7 @@ func (c *connection) ingestFile(ctx context.Context, env *connectors.Env, source
 	if err != nil {
 		return err
 	}
-	if err = rows.Close(); err != nil {
+	if err := rows.Close(); err != nil {
 		return err
```

Comment:
This triggers `revive`'s `if-return` – does it maybe feel like it could hurt readability? Feeling ambivalent about it

Reply:
this is `shadow` of `govet` looks like we should ignore this for readability.

Reply:
For `This triggers revive's if-return` this we can ignore `if-return` we have to choose either of both